backend/src/components/secureExchange.js

```diff
@@ -358,6 +374,40 @@ async function getExchange(req, res) {
     });
 }
 
+async function getExchangesCount(req, res) {
+  const token = getAccessToken(req);
+  if (!token && req.session.userMinCodes) {
+    return res.status(HttpStatus.UNAUTHORIZED).json({
+      message: 'No access token'
+    });
+  }
+  return Promise.all([
+    getCodeTable(token, CACHE_KEYS.EDX_SECURE_EXCHANGE_STATUS, config.get('edx:exchangeStatusesURL')),
+    getCodeTable(token, CACHE_KEYS.EDX_MINISTRY_TEAMS, config.get('edx:ministryTeamURL')),
+    getExchangesCountPaginated(req)
+  ])
+    .then(async ([statusCodeResponse, ministryTeamCodeResponse, dataResponse]) => {
+      let urExchangeCount = 0;
+      let exchangeCount = 0;
+      if (statusCodeResponse && ministryTeamCodeResponse && dataResponse?.content) {
```

Comment:
Is the count available in the payload returned to the paginated endpoint? It should be.